bin/GameIndex.dbf

```diff
@@ -1802,6 +1834,7 @@ XgKickHack = 1 // SPS.
 Serial = SCES-51685
 Name   = Primal
 Region = PAL-E-R
+vuRoundMode = 0 // Fix SPS (probably needs patch later)
```

Comment:
Why haven't you added the patch by NafanYa?
https://forums.pcsx2.net/Thread-Fixing-unplayable-games?pid=608669#pid608669

Reply:
Well seems ive overlooked some.